src/components/access-code/example-access-code-error.njk

```diff
@@ -58,30 +58,21 @@ layout: ~
         "classes": "ons-u-mb-xl"
     }) }}
 
-    <h2>If you don’t have an access code</h2>
+    <h2>If you do not have an access code</h2>
     <p>
-        You can <a href="#0">request a new access code</a> to start a new study if you have lost or not received an access code. This can be sent to you by text or post.
+        If you have lost or not received an access code, you can <a href="#0">request a new access code</a>.
+        This can be sent to you by text or post.
     </p>
 
     {% set content %}
-        <p>If you need to answer separately from the people you live with, you can <a href='#0'>request an individual access code</a> to start a separate study.</p>
+        <p>If you need to answer separately from the people you live with, you can <a href="#0">request an individual access code</a>.</p>
         {% call onsPanel({
                 "type": "warn"
             })
         %}
-            Someone in your household must still complete a study using a household access code
+            Someone in your household must still complete a study using this household access code
         {% endcall %}
     {% endset %}
 
-    {% call onsDetails({
-            "id": "details-access-code-error",
-            "title": "Need to answer separately from your household?",
-            "button": {
-                "close": "Hide this",
-                "contextSuffix": "content"
-            }
-        })
-    %}
-        {{ content | safe }}
-    {% endcall %}
+
```

Comment:
We don't want to remove the details component completely, just the button. Same with the other example

```suggestion
{% call onsDetails({
            "id": "details-access-code-error",
            "title": "Need to answer separately from your household?"
        })
    %}
        {{ content | safe }}
    {% endcall %}
```